action.yml

```diff
@@ -9,6 +9,14 @@ inputs:
     default: ""
     description: "Colcon mixin to be used to compile and test (empty means no mixin)"
     required: false
+  colcon-mixin-name-build:
+    default: ""
+    description: "Colcon mixin to be used to as input to colcon build(empty means no mixin). Overrides colcon-mixin-name"
+    required: false
+  colcon-mixin-name-test:
+    default: ""
+    description: "Colcon mixin to be used to as input to colcon test(empty means no mixin). Overrides colcon-mixin-name"
+    required: false
```

Comment:
It wouldn't be backwards compatible, but maybe `colcon-mixin-name` should just be removed in this case.

Reply:
yea it'd be a breaking change; i'm content with leaving it be for now.

Reply:
I think either way should be fine. We can always remove `colcon-mixin-name` later.

I'll let the people who actually have approval rights/write access take it from here and decide.